min_k8s_version/Tiltfile

```diff
@@ -0,0 +1,26 @@
+def min_k8s_version(min_version):
+
+   #This relies on the fact that the shell command returning something in the form 'gitVersion: vX.X.X'
+    version_str =  str(local('kubectl version --client  -o yaml| grep "gitVersion"', quiet=True))
```

Comment:
i think you want the server version? --client means you're just getting the kubectl version

Reply:
Oh right, I'll fix that!

Reply:
Fixed in 24cf22fbc413e681e6f928bf2a177c2ea23b4f1c !